src/modules/modules.controller.ts

```diff
@@ -1,11 +1,49 @@
-import { Controller } from '@nestjs/common';
+import { Body, Controller, Delete, Get, Param, Post, Put, Request } from '@nestjs/common';
 import { ApiTags } from '@nestjs/swagger';
+import { RequestWithUser } from 'src/auth/interfaces.interface';
 import { Auth } from 'src/common/decorator/auth.decorator';
+import { ApiCommonResponse } from 'src/common/decorator/commonApi.decorator';
+import { CreateModuleDTO } from './dto/create.module.dto';
+import { ModuleDTO } from './dto/module.dto';
 import { ModulesService } from './modules.service';
 
 @Auth()
 @ApiTags('modules')
 @Controller('/api/v1/module')
 export class ModulesController {
   constructor(private modulesService: ModulesService) {}
+
+  @Auth()
```

Comment:
As you already added Auth() in line 10 there should be no need on each method to use it.

Reply:
fixed